GW2EIEvtcParser/EIData/InstantCastFinders/MinionCastCastFinder.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Linq;
+using GW2EIEvtcParser.ParsedData;
+
+namespace GW2EIEvtcParser.EIData
+{
+    internal class MinionCastCastFinder : CheckedCastFinder<AnimatedCastEvent>
```

Comment:
Regarding this class, are we sure that a minion casting something necessarily matches the moment the command is given? I'm guessing you want to use this for the Jade Mech?

Reply:
No, it will not necessarily match the moment the command is given. The idea was to use this potentially for some of the Mech & Untamed Pet skills. I believe for those it should be good enough, since they are very close to actually pressing the skill for the Mech/Pet and can even queue up.